docs/src/main/sphinx/admin/resource-groups.md

```diff
@@ -171,15 +171,18 @@ documentation](https://docs.oracle.com/en/java/javase/22/docs/api/java.base/java
 
 - `queryType` (optional): string to match against the type of the query submitted:
 
-  - `SELECT`: `SELECT` queries.
-  - `EXPLAIN`: `EXPLAIN` queries (but not `EXPLAIN ANALYZE`).
-  - `DESCRIBE`: `DESCRIBE`, `DESCRIBE INPUT`, `DESCRIBE OUTPUT`, and `SHOW` queries.
-  - `INSERT`: `INSERT`, `CREATE TABLE AS`, and `REFRESH MATERIALIZED VIEW` queries.
-  - `UPDATE`: `UPDATE` queries.
-  - `DELETE`: `DELETE` queries.
-  - `ANALYZE`: `ANALYZE` queries.
+  - `SELECT`: [SELECT](/sql/select) queries.
+  - `EXPLAIN`: [EXPLAIN](/sql/explain) queries (but not [EXPLAIN ANALYZE](/sql/explain-analyze)).
+  - `DESCRIBE`: [DESCRIBE](/sql/describe), [DESCRIBE INPUT](/sql/describe-input), [DESCRIBE OUTPUT](/sql/describe-output), and `SHOW` queries.
+  - `INSERT`: [INSERT](/sql/insert), [CREATE TABLE AS](/sql/create-table-as), and [REFRESH MATERIALIZED VIEW](/sql/refresh-materialized-view) queries.
+  - `UPDATE`: [UPDATE](/sql/update) queries.
+  - `DELETE`: [DELETE](/sql/delete) queries.
+  - `ANALYZE`: [ANALYZE](/sql/analyze) queries.
   - `DATA_DEFINITION`: Queries that alter/create/drop the metadata of schemas/tables/views,
```

Comment:
This is pretty vague and ideally we add some example and avoid the /. Maybe something like this

Queries that affect the data definition. These include CREATE, ALTER, and DROP statements for schemas, tables, views, and materialized views, as well as statements that ...